vector/src/main/res/values/strings.xml

```diff
@@ -590,24 +590,24 @@
     <string name="settings_notification_privacy_nosecure_message_content">• Notifications contain <b>meta and message data</b></string>
     <string name="settings_notification_privacy_message_content_not_shown">• Notifications will <b>not show message content</b></string>
 
-    <string name="settings_notification_ringtone">Notification sound</string>
+    <string name="settings_notification_ringtone">Notification Sound</string>
     <string name="settings_enable_all_notif">Enable notifications for this account</string>
     <string name="settings_enable_this_device">Enable notifications for this device</string>
     <string name="settings_turn_screen_on">Turn the screen on for 3 seconds</string>
 
-    <string name="settings_containing_my_display_name">Msgs containing my display name</string>
-    <string name="settings_containing_my_user_name">Msgs containing my user name</string>
-    <string name="settings_messages_in_one_to_one">Msgs in one-to-one chats</string>
-    <string name="settings_messages_in_group_chat">Msgs in group chats</string>
-    <string name="settings_invited_to_room">When i\'m invited to a room</string>
-    <string name="settings_call_invitations">Call invitations</string>
-    <string name="settings_messages_sent_by_bot">Messages sent by bot</string>
+    <string name="settings_containing_my_display_name">Display Name Mentions</string>
```

Comment:
I do not like `Msgs` but your new wording are less accurate than the previous one. Consider restoring the previous strings for all this section

Reply:
I agree with the use of `msgs` but I thought these changes were pretty accurate. The only one that I felt was a bit of a jump was `bot messages` but `room invitations` seems like the exact same meaning as before. If that doesn't make sense then would `call invitations` also be too vague? As for the other instances, I think the term `mention` is used in other places in the app to refer to the same thing. I could change `One-to-One Chats` to `One-to-One Chat Messages` and the same for groups but I think the use of `chat` is enough to explain the meaning.

Reply:
The aim of this section is to set up the notification when the received message match some rules. That's why I think `Message containing my display name` is better than `Display name mention`.